.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Pull request checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
+- [ ] Build (`npm run build`) was run locally for affected output targets
+
+## Pull request type
+
+<!-- Please do not submit updates to dependencies unless it fixes an issue. -->
+
+<!-- Please try to limit your pull request to one type, submit multiple pull requests if needed. -->
+
+Please check the type of change your PR introduces:
+
+- [ ] Bugfix
+- [ ] Feature
+- [ ] Code style update (formatting, renaming)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] Documentation content changes
+- [ ] Other (please describe):
+
+## What is the current behavior?
+
+<!-- Please describe the current behavior that you are modifying. -->
+
+<!-- Issues are required for both bug fixes and features. -->
+
+Issue #:
+
+## What is the new behavior?
+
+<!-- Please describe the behavior or changes that are being added by this PR. -->
+
+-
+-
+-
+
+## Does this introduce a breaking change?
+
+- [ ] Yes
+- [ ] No
+
+<!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->
+
```

Comment:
Should we ask folks to tell us how they tested the functionality they're proposing? Unit tests can get us there to a certain extent, but I think (for now) a test section would be equally helpful to understand where they're coming from when they submit a PR:

```

## Testing

<!-- Please describe the steps you took to test the changes in this PR. These steps can be programmatic (e.g. unit tests) and/or manual. -->'
```

Reply:
I think community PRs (the issue attached) should require a reproduction app to validate against. We can then install the dev build for the PR and run the build output to verify anything beyond unit test coverage. This is another thing we do in Framework that works well. 

This reduces the need to test PRs in different ways, based on whatever steps a contributor outlines. 

Thoughts?

Reply:
I like it 👍